jdaviz/configs/mosviz/plugins/parsers.py

```diff
@@ -644,13 +644,10 @@ def mos_niriss_parser(app, data_dir, obs_label=""):
     with app.data_collection.delay_link_manager_update():
 
         for label, data in add_to_glue.items():
-            app.data_collection[label] = data
+            app.add_data(data, label, notify_done=False)
 
         # We then populate the table inside this context manager as _add_to_table
         # does operations that also trigger link manager updates.
-
-        print("Populating table")
```

Comment:
I thought @javerbukh wanted to keep these `print` functions when I suggested to remove them in a different PR? 🤷

Reply:
Oh, I wasn't aware of that conversation, I thought they were prints for debugging that snuck into main.

Reply:
@javerbukh confirmed in our dev meeting that these were relics from when loading took a very long time and are irrelevant now. He's ok with removing them.